test/smoke/dapi.js

```diff
@@ -1,10 +1,58 @@
+const DAPIClient = require('@dashevo/dapi-client');
+const RpcClient = require('@dashevo/dashd-rpc/promise');
+const getNetworkConfig = require('../../lib/test/getNetworkConfig');
+
+const networkConfig = getNetworkConfig();
+
+const fetch = require('node-fetch');
+
 describe('DAPI', () => {
-  // getBlockHash height:1
-  it('should respond data from chain');
+  networkConfig.inventory.masternodes.hosts.forEach((nodeName) => {
+    describe(nodeName, () => {
+      const options = {
+        debug: false,
+        verbose: false,
+        errors: false,
+        warnings: false,
+        seeds: [{ ip: networkConfig.inventory._meta.hostvars[nodeName].public_ip }],
+      };
+      const dapi = new DAPIClient(options, 3000);
```

Comment:
We can't use DAPI client here :(. Please use jayson.

Reply:
moved on jayson